src/v/cluster/shard_placement_table.cc

```diff
@@ -365,6 +370,9 @@ ss::future<> shard_placement_table::initialize_from_kvstore(
       [&ntp2init_data](shard_placement_table& spt) {
           return spt.scatter_init_data(ntp2init_data);
       });
+    for (auto& spt : extra_spts) {
+        co_await spt->scatter_init_data(ntp2init_data);
+    }
```

Comment:
any reason why we process existing shard data concurrently, but extra shards one by one?